tests/e2e_regression/expected/db__sdk_balances.csv

```diff
@@ -80,7 +80,7 @@ emerald,oasis1qz629557xs0j6vk9ulwk4k3mjkdpvfncrs5h0zg8,ROSE,8591305825000000000
 emerald,oasis1qz670t637yyxshnlxhjj5074wgwl94d0x5x69zqd,ROSE,34225830891942957224
 emerald,oasis1qz6m48jljzlpdz52prgfhrppnz0ffshlt5j7kjaz,ROSE,2906029534421611553
 emerald,oasis1qz72lvk2jchk0fjrz7u2swpazj3t5p0edsdv7sf8,ROSE,411567692021454489
-emerald,oasis1qz78phkdan64g040cvqvqpwkplfqf6tj6uwcsh30,ROSE,131
+emerald,oasis1qz78phkdan64g040cvqvqpwkplfqf6tj6uwcsh30,ROSE,121166699302776158
```

Comment:
**NOTE:** This is the fee accumulator addr. **The diff is expected** because the previously, slow-sync (under which regime the whole e2e regression runs) started with the (known incorrect) assumption that runtime balances are all 0. Now, the `finalizeFastSync()` method runs first (even though fast-sync itself doesnt; that method is really more of a `preparePreconditionsForSlowSync()`) and pulls the initial balance from the node.

(*) Although in